js/components/widgets/plot/controllers/PlotsController.js

```diff
@@ -197,23 +197,19 @@ define(function (require) {
         if (inst != undefined && inst.getTimeSeries() != undefined) {
```

Comment:
@filippomc what are these changes for? they seem to impact the old widget controller rather than the new widget which shouldnt use a controller - correct me if I'm wrong.

Reply:
This is coming from Matt's changes for OSB.The history is he developed Plot.js to use as a widget but it couldn't be transformed to a plain component so I created PlotComponent.js